qutebrowser/mainwindow/statusbar/bar.py

```diff
@@ -124,14 +142,54 @@ class StatusBar(QWidget):
             {{ color['statusbar.bg.error'] }}
         }
 
+        QWidget#StatusBar[severity="error"] QLabel {
+            {{ color['statusbar.fg.error'] }}
+        }
+
         QWidget#StatusBar[severity="warning"] {
             {{ color['statusbar.bg.warning'] }}
         }
 
+        QWidget#StatusBar[severity="warning"] QLabel {
+            {{ color['statusbar.fg.warning'] }}
+        }
+
+        QWidget#StatusBar[prompt_active="true"] {
+            {{ color['statusbar.bg.prompt'] }}
+        }
+
+        QWidget#StatusBar[prompt_active="true"] QLabel {
+            {{ color['statusbar.fg.prompt'] }}
+        }
+
+        QWidget#StatusBar[prompt_active="true"] QLineEdit {
+            {{ color['statusbar.fg.prompt'] }}
+        }
+
+        QWidget#StatusBar[insert_active="true"] {
+            {{ color['statusbar.bg.insert'] }}
+        }
+
+        QWidget#StatusBar[insert_active="true"] QLabel {
+            {{ color['statusbar.fg.insert'] }}
+        }
+
+        QWidget#StatusBar[command_active="true"] {
+            {{ color['statusbar.bg.command'] }}
+        }
+
+        QWidget#StatusBar[command_active="true"] QLabel {
+            {{ color['statusbar.fg.command'] }}
+        }
+
+        QWidget#StatusBar[command_active="true"] QLineEdit {
+            {{ color['statusbar.fg.command'] }}
+        }
+
         QLabel, QLineEdit {
-            {{ color['statusbar.fg'] }}
             {{ font['statusbar'] }}
         }
+
     """
```

Comment:
For some entries here you can combine them too (like mentioned above) - also in some cases there is no `QLineEdit` entry, only a `QLabel` one - is that intentional?